app/src/router/request.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Src\router;
+
+class Request
+{
+    public $params;
+    public $reqMethod;
+    public $contentType;
+    private $user;
+
+    public function __construct($params = [])
```

Comment:
@jjsimonAB its better for services to avoid calling global variables, they have to ba pass or use a DI to access this